applications/machine_learning/configuration/thingy53_nrf5340_cpuapp/app_ZDebug.conf

```diff
@@ -104,6 +104,9 @@ CONFIG_MCUMGR_SMP_BT_AUTHEN=n
 CONFIG_MCUMGR_CMD_IMG_MGMT=y
 CONFIG_MCUMGR_CMD_OS_MGMT=y
 
+# mcumgr buffer size must be increased to prevent MGMT_ERR_ENOMEM.
+CONFIG_MCUMGR_BUF_SIZE=1024
```

Comment:
I am working on set of macros that will calculate the buffer by the description of frames that need to fit into it.
But the size of the buffer itself depends on many different thins, for example if user enables shell then the size will determine how much shell output will be returned.
When user defined frames this may be even bigger. Making that default dependent on the number of images will not fix it.
This is like heap or stack size, there is not good dependency for it.

Reply:
For example: the mcuboot uses similar buffer and returns similar CBOR encoded data, but does not return everything, so a buffer, for the same purpose, may be smaller

Reply:
and when that is in place and ready to be rolled out, then that can be done seamlessly to the users if we change the default in `Kconfig.nrf`.

If we keep the PR as is, with  `CONFIG_MCUMGR_BUF_SIZE=1024` in n-number of conf files, then we need to update ALL those places later.

Reply:
Ok, will do